src/main/java/org/ships/vessel/sign/EOTSign.java

```diff
@@ -82,7 +82,7 @@ public boolean onSecondClick(@NotNull LivePlayer player, SyncBlockPosition posit
         }
         LiveSignTileEntity stes = (LiveSignTileEntity) lte;
         new ShipsUpdateBlockLoader(position).loadOvertime(vessel -> {
-            if (stes.getTextAt(1).isPresent() && stes.getTextAt(1).get().contains(AText.ofPlain("{"))) {
+            if (stes.getTextAt(1).isPresent() && stes.getTextAt(1).get().toPlain().contains("{")) {
```

Comment:
I've tested it on my Paper server and the previous solution didn't work... This one does